modules/javafx.graphics/src/main/java/com/sun/glass/ui/win/WinAccessible.java

```diff
@@ -1562,6 +1584,9 @@ private void Toggle() {
 
     private int get_ToggleState() {
         if (isDisposed()) return 0;
+        if (getAttribute(ROLE) == AccessibleRole.CHECK_BOX_TREE_ITEM) {
+            return (int)getAttribute(TOGGLE_STATE);
```

Comment:
I recommend mapping the return value of `getAttribute(TOGGLE_STATE)`, which should be a platform-independent value, to one of the three Windows-specific values. Otherwise you are making an assumption that might not hold for other platforms (e.g., macOS).

Reply:
Added a new enum `ToggleState`.